fedimint-core/src/envs.rs

```diff
@@ -2,6 +2,15 @@
 // introducing extra dependencies between core and build modules.
 pub const FEDIMINT_BUILD_CODE_VERSION_ENV: &str = "FEDIMINT_BUILD_CODE_VERSION";
 
+/// In tests we want to routinely enable an extra unknown module to ensure
+/// all client code handles correct modules that client doesn't know about.
+pub const FM_USE_UNKNOWN_MODULE_ENV: &str = "FM_USE_UNKNOWN_MODULE";
+
+/// Check if env variable is set and not equal `0` or `false` which are common
+/// ways to disable something.
+pub fn is_env_var_set(var: &str) -> bool {
```

Comment:
If you make changes, might be able to sneak this in https://github.com/fedimint/fedimint/pull/4247#discussion_r1500582466

No biggie, I can also address in a followup.